Mathlib/Order/InitialSeg.lean

```diff
@@ -9,16 +9,20 @@ import Mathlib.Order.WellFounded
 /-!
 # Initial and principal segments
 
-This file defines initial and principal segments.
+This file defines initial and principal segment embeddings. Though these definitions make sense for
+arbitrary relations, they're intended for use with well orders.
+
+An initial segment within a well order is any downwards closed subset. A principal segment is an
+initial segment other than `Set.univ`.
```

Comment:
Your sentence is technically true but misleading. Instead of explaining what the definitions are equivalent to in the well-order case, it should give the general definitions, namely
```suggestion
An initial segment is simply a lower set. A principal segment is a set of the form `Set.Iic x` for some `x`.
```

Reply:
`Set.Iio`, not `Set.Iic`. I've updated the descriptions of `InitialSeg` and `PrincipalSeg` with your suggestion in mind.